azext_edge/tests/edge/orchestration/resources/test_instances_unit.py

```diff
@@ -411,3 +449,230 @@ def test_parse_feature_kvp_nargs(feature_scenario: Optional[dict]):
 
     result = parse_feature_kvp_nargs(features=feature_scenario.get("inputs"), **kwargs)
     assert result == feature_scenario.get("expected", {}), f"Expectation failure for: {feature_scenario.get('case')}"
+
+
+@pytest.mark.parametrize(
+    "spc_name",
+    [None, generate_random_string()],
+)
+@pytest.mark.parametrize(
+    "skip_role_assignments",
+    [None],
```

Comment:
if these parameters(and below) only have one value, should we assign them within the test function instead?

Reply:
They will have more, the reason it is like this is the test is a TODO.

Reply:
ok that make sense